### PR TITLE
feat:json extractor

### PR DESCRIPTION
**Pull request by carlbrugger**

### CODE REVIEW

File: plugins/json-extractor/package.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@flatfile/plugin-json-extractor",
+  "version": "1.0.0",
+  "description": "A plugin for parsing json files in Flatfile.",
+  "registryMetadata": {
+    "category": "extractors"
+  },
+  "engines": {
+    "node": ">= 12"
```
Comment (hansjhoffman) [inline plugins/json-extractor/package.json:9]: 🔧 I think we decided on `16` here

**Comment by carlbrugger** [comment]
Thanks for the feedback @hansjhoffman! 🌮

File: plugins/json-extractor/package.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@flatfile/plugin-json-extractor",
+  "version": "1.0.0",
```
Comment (ashleygmulligan2) [inline plugins/json-extractor/package.json:3]: let's do 0.0.1 for first release

**Review by ashleygmulligan2 — CHANGES_REQUESTED** [review]
Run npm run changeset from root

**Review by ashleygmulligan2 — APPROVED** [review]

**Pull request merged**